oioioi/base/registration_backend.py

```diff
@@ -63,18 +63,18 @@ def register(self, form):
 
 
 urlpatterns = [
-    re_path(r'^register/$', RegistrationView.as_view(), name='registration_register'),
+    re_path(r'^sign-up/$', RegistrationView.as_view(), name='sign-up'),
 ]
 
 if not settings.SEND_USER_ACTIVATION_EMAIL:
     urlpatterns += [
         re_path(
-            r'^register/complete/$',
+            r'^sign-up/complete/$',
```

Comment:
I would like to see some redirection from previous register to sign-up just not to broke already existing links in some content